main.lua

```diff
@@ -770,7 +780,7 @@ function love.run()
 			host_time=host_time+dt
 			if paused then
 			else
-				if love.update then love.update(frametime) end -- will pass 0 if love.timer is disabled
+				if love.update then love.update(dt) end -- will pass 0 if love.timer is disabled
```

Comment:
`frametime` is correctly being passed to love.update, as love.update is being called every frametime (i.e; if `dt>frametime`, love.update will be called `dt//frametime` times.

Reply:
Ok, this was a bit puzzling because there’s a comment saying dt should be passed instead and I was having timing problems, maybe this isn’t necessary though and just the pico8.frame thing is enough but I need to retest when I’m home

Reply:
I double checked this too, if i keep frame time I still get missing frames in the intro, while they are resolved if I use dt here. From what I see, all 3 changes are necessary to have correct timing when you reload a cart, otherwise soem frames are skipped. See if you can test it or I can also prepare a demo cart to show it clealry

Reply:
The confusing comment originates from LOVE itself (the `love.run` function is copied from LOVE2D and replicated here). In LOVE, `dt` would indeed be passed to `love.update`, but we ensure 30/60 FPS, so we run the frame with a fixed delta of `frametime` as many times as we have to in order to work off our backlog in `dt`.